### PR TITLE
Story/epgr 2

### PR DESCRIPTION
**Pull request by Mihika0409**

### CODE REVIEW

**Comment by jdamerow** [comment]
@Mihika0409 as I said before, please resolve conflicts before reopening the PR (and then reopen it, don't create a new one).

File: grazer/pom.xml [hunk 1]
```diff
@@ -215,6 +215,19 @@
...
+		
+		<!-- Hibernate and Dependent jars -->
+		<!-- Hibernate core -->
+		<dependency>
+			<groupId>org.hibernate</groupId>
+			<artifactId>hibernate-core</artifactId>
+			<version>5.2.10.Final</version>
+		</dependency>
+
+
```
Comment (jdamerow) [inline grazer/pom.xml:230]: does this belong to this ticket?

File: grazer/src/main/webapp/WEB-INF/views/person.jsp [hunk 2]
```diff
@@ -29,8 +27,53 @@
...
+</c:if>
+<c:if test="${not empty edge.endTime}">
+    ending in ${edge.endTime}
+</c:if>
+<c:if test="${not empty edge.occurred}">
+    in ${edge.occurred}
+</c:if>
+    found in <a href="${edge.sourceUri}" target="_blank"><i class="fa fa-globe" aria-hidden="true"></i> ${edge.sourceUri}</a>.
+</li>
+</c:forEach>
```
Comment (jdamerow) [inline grazer/src/main/webapp/WEB-INF/views/person.jsp:78]: this is the old code. Please don't override the new code with the old one.

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: grazer/pom.xml (hunk 1 above)
Comment (Mihika0409) [reply grazer/pom.xml:230]: I had accidentally written this into this ticket, and didn't delete it. I moved this into EPGR-3

**Pull request closed without merging**